src/Microsoft.Azure.SignalR.Management/Configuration/ContextOptions.cs

```diff
@@ -12,14 +12,25 @@ internal class ContextOptions
         //Users not allowed to configure it
         internal string ProductInfo { get; set; }
 
+        /// <summary>
+        /// Gets or sets the service endpoints.
+        /// </summary>
         public ServiceEndpoint[] ServiceEndpoints { get; set; }
 
+        /// <summary>
+        /// Gets or sets the ApplicationName which will be prefixed to each hub name
+        /// </summary>
         public string ApplicationName { get; set; }
 
+        /// <summary>
+        /// Gets or sets the total number of connections from SDK to Azure SignalR Service. Default value is 1.
+        /// </summary>
         public int ConnectionCount { get; set; }
 
+        /// <summary>
+        /// Gets or sets the proxy used when ServiceManager will attempt to connect to Azure SignalR Service.
```

Comment:
`will attempt` => `attempts`

Reply:
`ServiceManager` => ` instance of <see cref="IServiceManager"/>`